web/package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prettier":"prettier --write 'src/**/*.{scss,tsx,js,ts,x}'"
```

Comment:
I've added the prettier command to be easier to run it locally. Let me know if I should do it differently.

Reply:
sure this is fine but remove scss, js, and x.